src/config/dark-sites.config

```diff
@@ -398,6 +406,7 @@ marekmaskarinec.github.io
 maroon.jonah.pw
 marte.dev
 matteotiscia.com
+mau.fi
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/105616364-41971a80-5dce-11eb-99b1-937270062523.png)

Reply:
This one is weird, tried visiting the site from 5 other browsers and it was always dark themed!
Turns out that it depends on your browser's appearance option, if it was on dark then it should been displayed as dark themed site.

Reply:
> Turns out that it depends on your browser's appearance option, if it was on dark then it should been displayed as dark themed site.

And the sites inside dark-sites.config needs to be by default regardless of appearance option have a dark theme.